images/base/Dockerfile

```diff
@@ -23,7 +23,7 @@ FROM ubuntu:19.10
 
 # Configure containerd and runc binaries from kind-ci/containerd-nightlies repository
 # The repository contains latest stable releases and nightlies built for multiple architectures
-ARG CONTAINERD_VERSION="v1.3.0-27-g54658b88"
+ARG CONTAINERD_VERSION="v1.3.2"
```

Comment:
We are downloading containerd from our nighlies repository, can check from my mobile if we are releasing this version

Reply:
Give me a little credit :-)

Reply:
Not only do I actually check the releases when doing this, I bumped the base image to include it, which won't build if the release doesn't exist 🙃

Reply:
:man_shrugging: holiday are starting to affect my brain :upside_down_face: 
Sorry

Reply:
I was doubting if it was correct because I was missing the short hash appended to the version, I've forgotten that we have 2 jobs in travis to release nightlies, and one is publishing containerd releases without the commit hash appended :sweat_smile: